src/dmd/clone.d

```diff
@@ -185,31 +185,70 @@ private bool needOpAssign(StructDeclaration sd)
 }
 
 /******************************************
- * Build opAssign for struct.
- *      ref S opAssign(S s) { ... }
+ * Build opAssign for a `struct`.
  *
- * Note that `s` will be constructed onto the stack, and probably
- * copy-constructed in caller site.
+ * The generated `opAssign` function has the following signature:
+ *-----
+ *      ref S opAssign(S s)    // S is the name of the `struct`
```

Comment:
1. Please don't indent the code in the example
2. Only use three dashes for the code block

Reply:
Done. Thanks!